vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -1,301 +1,135 @@
+              name="project"
+              rows="1"
+              autocomplete="given-project-info"
+              v-model="projectInfo"
               :disabled="disableInput"
+              placeholder="Tell us about your project"
             ></textarea>
           </div>
-          <div>
-            <div v-if="contactType < 2" class="radio-buttons mb-3">
-              <label class="contactLabel1">
-                <input
+          <div class="tw-pt-5 lg:tw-pt-10">
+            <input
+              class="custom-text-input v2-title-2-text"
+              type="text"
+              name="reference"
+              required
+              autocomplete="given-reference"
+              v-model="reference"
+              :disabled="disableInput"
+              placeholder="How did you find us?"
+            />
+            <span v-if="showValidationError" class="error tw-text-red-600"
+              >This field is required</span
+            >
+          </div>
+
+          <div class="tw-py-16">
+            <div class="tw-text-slate-400 v2-title-2-text">
+              What's your preferred mode of communication?
+            </div>
+            <div class="tw-flex tw-flex-col tw-w-fit sm:tw-flex-row tw-pt-6">
+              <label class="contactLabel1"
+                ><input
+                  class="tw-hidden"
                   type="radio"
                   name="contact_type"
-                  value="0"
+                  value="2"
                   v-model.number="contactType"
                   :disabled="disableInput"
                 />
-                <div class="box">
-                  <span class="contact-form-text"
-                    >If you love to talk, You can schedule a meeting with our
-                    team member according to your convenience from here.</span
-                  >
-                </div>
-              </label>
-              <label class="contactLabel2">
-                <input
+                <div class="contact-button tw-mb-6 sm:tw-mr-9 sm:tw-mb-0">
+                  <font-awesome-icon
+                    class="fa -tw-rotate-45 tw-w-6 tw-h-6 tw-text-black-900"
+                    :icon="phone"
+                    aria-hidden="true"
+                  />
+                  <span class="tw-ml-5 v2-normal-3-text">Call</span>
+                </div></label
+              >
+
+              <label class="contactLabel2"
+                ><input
+                  class="tw-hidden"
                   type="radio"
                   name="contact_type"
                   value="1"
                   v-model.number="contactType"
                   :disabled="disableInput"
                 />
-                <div class="box">
-                  <span class="contact-form-text"
-                    >If you prefer chat or email, please click here.</span
-                  >
-                </div>
-              </label>
+                <div class="v2-normal-3-text contact-button">
```

Comment:
What will happen if i click on this?

Reply:
We click on this button that send the just mail with `thank you popup `

Reply:
Its not showing me any popup, when click

Reply:
This functionality is same as our live contact page